pavelib/prereqs.py

```diff
@@ -11,6 +11,10 @@
 
 PREREQS_MD5_DIR = os.getenv('PREREQ_CACHE_DIR', Env.REPO_ROOT / '.prereqs_cache')
 NPM_REGISTRY = "http://registry.npmjs.org/"
+
+# If you make any changes to this list you also need to make
+# a corresponding change to circle.yml, which is how the python
```

Comment:
Wait, no it's right - the prereqs are defined/installed via the dependencies override section of circle.yml.